### ISSUE TITLE
这个aspp输出x未定义

### ISSUE THREAD

**Issue by sansuitaibai** [YangParky/BASeg#8]
# ASPP
        x = self.ASPP(x)
        x = torch.cat([x, f4], dim=1)

**Comment by YangParky** [comment]
感谢指出，已修改
